frontend/src/app/modules/boards/board/board-list/board-list.component.ts

```diff
@@ -393,7 +391,10 @@ export class BoardListComponent extends AbstractWidgetComponent implements OnIni
     let observable = this
       .apiv3Service
       .queries
-      .find(this.columnsQueryProps, this.queryId);
+      .find(this.columnsQueryProps, this.queryId)
+      .pipe(
+        retry(3)
```

Comment:
This retry doesn't decrease the probability of a 429 but rather increases it, since you'll just be refreshing the cooldown. However, if you feel it's necessary we can keep it in.

Reply:
Hi @b12f ,

Thanks for your feedback.

Markus suggested that just retrying would be enough and that we wouldn't need to delay the calls. This makes sense to me. For example, the trial backend actually supports 20 requests per second. If we reach the limit by sending for example 30 requests, then we would retry 10 at the most, so they wouldn't fail again. It is not my domain, I don't understand how does this increases the probability of a 429. Could you elaborate more on this, please?